### PR TITLE
Ovens now have smoke particles when running

### PR DESCRIPTION
**Pull request by MaxIsJoe**
parity feature with SS13

CL: [Improvement] Ovens have a smoke particle effect when cooking.

### CODE REVIEW

**Comment by ThatDan123** [comment]
Particles arent networked, just move the on/off to one of those Sync functions that already exist

**Comment by MaxIsJoe** [comment]
> Particles arent networked, just move the on/off to one of those Sync functions that already exist

Done 👍

File: UnityProject/Assets/Scripts/Objects/Machines/Oven.cs [hunk 1]
```diff
@@ -388,6 +392,7 @@ public OvenIdle(Oven oven)
 				oven.OnSyncOvenGlow(oven.ovenGlowEnabled, false);
 				oven.HaltOven(true);
 				oven.SetWattage(oven.circuitWattage);
+				oven.particles.Stop();
```
Comment (ThatDan123) [inline UnityProject/Assets/Scripts/Objects/Machines/Oven.cs]:
```suggestion
```

Comment (MaxIsJoe) [reply UnityProject/Assets/Scripts/Objects/Machines/Oven.cs]: merged

**Review by ThatDan123 — APPROVED** [review]

**Review by Bod9001 — APPROVED** [review]

**Pull request merged**